dropwizard-auth/src/test/java/io/dropwizard/auth/CachingAuthorizerTest.java

```diff
@@ -119,8 +125,17 @@ public void calculatesTheSizeOfTheCache() throws Exception {
 
     @Test
     public void calculatesCacheStats() throws Exception {
-        cached.authorize(principal, role);
         assertThat(cached.stats().loadCount()).isEqualTo(0);
+        cached.authorize(principal, role);
+        assertThat(cached.stats().loadCount()).isEqualTo(1);
         assertThat(cached.size()).isEqualTo(1);
     }
+
+    @Test
+    public void shouldPropagateRuntimeException() throws AuthenticationException {
+        final RuntimeException e = new NullPointerException();
+        when(underlying.authorize(principal, role)).thenThrow(e);
+        expected.expect(CoreMatchers.sameInstance(e));
```

Comment:
Oh that's nice, I didn't even know it existed! I'll change this and update the PR.